data/app-schema-tutorial/styles/default_line.sld

```diff
@@ -12,7 +12,7 @@
 		<UserStyle>
 		    <!-- they have names, titles and abstracts -->
 		  
-			<Title>A boring default style</Title>
+			<Title>Green Line</Title>
```

Comment:
Consistent title case between "Green Line" vs "Red Square point" etc... do you care?

Reply:
Is it just me or the default line is blue, not green? :-D

Reply:
I will fix that